README.md

```diff
@@ -27,6 +27,17 @@ This boilerplate comes with batteries included, you’ll find:
 4. Create a new Git repository (`git init`)
 5. Create the initial Git commit (`git commit -a -m "Initial commit"`)
 
+## Development
+
+Before starting the server with `mix phx.server` or testing the OTP/Docker setup with `make dev-start`, generate secret values for the following keys in [`.env.dev`](./.env.dev):
+    * `SECRET_KEY_BASE`
+    * `SIGNING_SALT`
+
+```shell
+> mix phx.gen.secret
+Eif/HWo9DB0A8h+tYYJDZOZUKXVTzLnHa7faTFySn6eIqocYqbcE8Hs9C9qp98Po
+```
+
```

Comment:
@remiprev Do you think this be enough?

Reply:
This should be added to `BOILERPLATE_README.md`, not `README.md` 😄

I would also try to avoid listing specific commands and specific variables. We already have [this line](https://github.com/mirego/elixir-boilerplate/blob/master/BOILERPLATE_README.md#initial-setup): 

> Create both .env.dev.local and .env.test.local from empty values in .env.dev and .env.test

We should just add a small note mentioning the need to populate missing variables (instructions are already in the `.env.dev` file).